cmd/perfguard/cmd_lint.go

```diff
@@ -1,10 +1,14 @@
 package main
 
 import (
+	"errors"
 	"flag"
+	"fmt"
 	"io"
 )
 
+var ErrIssuesFound = errors.New("found issues")
```

Comment:
As a side note: I don't think that this variable needs to be exported (capitalized).